src/commands/xread.json

```diff
@@ -67,6 +67,33 @@
                     }
                 ]
             }
-        ]
+        ],
+        "reply_schema": {
```

Comment:
please add a description whenever you have a "type" (maybe not everywhere, but this is the rule of thumb)